daml/Common.daml

```diff
@@ -33,93 +38,81 @@ data CommonContracts = CommonContracts
 
   -- Identities
-  regulatorServiceOfferCid <- submit operator do exerciseCmd operatorServiceCid Operator.OfferRegulatorService with provider = operator; customer = custodian
-  regulatorServiceCid <- submit custodian do exerciseCmd regulatorServiceOfferCid Regulator.Accept
-  identityVerificationRequestCid <- submit custodian do exerciseCmd regulatorServiceCid Regulator.RequestIdentityVerification with legalName = "CusotidanLegalName"; location = "CustodianLocation"; observers = [public]
+  regulatorServiceOfferCid <- submit operator do exerciseCmd operatorServiceCid Operator.OfferRegulatorService with provider = operator; customer = bank
+  regulatorServiceCid <- submit bank do exerciseCmd regulatorServiceOfferCid Regulator.Accept
+  identityVerificationRequestCid <- submit bank do exerciseCmd regulatorServiceCid Regulator.RequestIdentityVerification with legalName = "BankLegalName"; location = "BankLocation"; observers = [public]
   verifiedIdentity <- submit operator do exerciseCmd regulatorServiceCid Regulator.VerifyIdentity with ..
   regulatorServiceOfferCid <- submit operator do exerciseCmd operatorServiceCid Operator.OfferRegulatorService with provider = operator; customer = exchange
   regulatorServiceCid <- submit exchange do exerciseCmd regulatorServiceOfferCid Regulator.Accept
   identityVerificationRequestCid <- submit exchange do exerciseCmd regulatorServiceCid Regulator.RequestIdentityVerification with legalName = "ExchangeLegalName"; location = "ExchangeLocation"; observers = [public]
   verifiedIdentity <- submit operator do exerciseCmd regulatorServiceCid Regulator.VerifyIdentity with ..
 
-  pure CommonContracts with ..
+  pure Providers with ..
 
-onboardCustomer : CommonContracts -> Text -> Asset -> Party -> Script CustomerContracts
-onboardCustomer CommonContracts{..} party asset accountNominee = do
+onboardCustomer : Providers -> Text -> Script Customer
+onboardCustomer Providers{..} party = do
   customer <- allocatePartyWithHint party $ PartyIdHint party
   let
-    tradingAccountId      = Id with signatories = fromList [ custodian, customer ]; label = "TradingAccount-" <> party; version = 0
-    tradingAccount        = Account with provider = custodian; owner = customer; id = tradingAccountId
-    allocationAccountId   = Id with signatories = fromList [ custodian, customer ]; label = "AllocationAccount-" <> party; version = 0
-    allocationAccount     = Account with provider = custodian; owner = customer; id = allocationAccountId
-    awayAccountId         = Id with signatories = fromList [ custodian, customer ]; label = "AwayAccount-" <> party; version = 0
-    awayAccount           = Account with provider = custodian; owner = customer; id = awayAccountId
-    safekeepingAccountId  = Id with signatories = fromList [ custodian, customer ]; label = "SafekeepingAccount-" <> party; version = 0
-    safekeepingAccount    = Account with provider = custodian; owner = customer; id = safekeepingAccountId
```

Comment:
You've removed these. Don't you need them?

Reply:
I use the mainAccount for the issuance now